node/clients/checkout.ts

```diff
@@ -43,13 +43,16 @@ export class Checkout extends JanusClient {
     return this.get<PaymentSession>(this.routes.getPaymentSession())
   }
 
-  public addItem = (orderFormId: string, items: any, salesChannel?: string) =>
-    this.post<CheckoutOrderForm>(
+  public addItem = (orderFormId: string, items: any, salesChannel?: string, allowOutdatedData?: any) =>
```

Comment:
did you forgot to type this as `string[]` or there is some reason to not type this?
```suggestion
  public addItem = (orderFormId: string, items: any, salesChannel?: string, allowOutdatedData?: string[]) =>
```